Source/Plugins/bsfOpenAudio/BsOAAudioSource.cpp

```diff
@@ -597,18 +623,20 @@ namespace bs
 		UINT32 sampleBufferSize = numSamples * (info.bitDepth / 8);
 
 		UINT8* samples = (UINT8*)bs_stack_alloc(sampleBufferSize);
+		std::fill(samples, samples + sampleBufferSize, 0);
```

Comment:
Ideally we can avoid doing a fill over the entire buffer. It wastes performance as in 99% of the cases it will just get overwritten.